Documentation/ceph-configuration.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Configuration
+weight: 3700
+indent: true
+---
+
+# Configuration
+
+For most any Ceph cluster, the user will want to -- and may need to -- change some Ceph
```

Comment:
nit: should the hyphens not have whitespace between the words? I'm confused by the spacing.

Reply:
I think by most US style guides, the em dash isn't separated by spaces, yes.